skrf/network.py

```diff
@@ -306,10 +300,18 @@ class Network:
         'time_impulse': None,
         'time_step': None,
     }
+
     """
     Component functions like 're', 'im', 'mag', 'db', etc.
     """
 
+    @classmethod
+    @lru_cache()
```

Comment:
For my knowledge: did you test if the use or lru_cache improved performance?

Reply:
No I hadn't. As the function is called only a few times during module initialization, it would not be much either.